dice1.js

```diff
@@ -0,0 +1,35 @@
+const rollDice = function(){
+    const MyRandom1 = Math.floor(Math.random() * 6);
```

Comment:
the dice roll range is from 0 to 5, dice rolls are from 1 to 6, think about adding 1.